### PR TITLE
Add customer name and timestamp to report name. 

### PR DESCRIPTION
**Pull request by awright18**
Adds customer name to the report name, and confirms running when reports are found in the working directory.

### CODE REVIEW

File: src/Tool/Commands/BaseCommand.cs [hunk 1]
```diff
@@ -9,25 +11,51 @@
...
+
+    bool ConfirmCreateNewReport()
+    {
+        return Confirm($"Existing throughput reports were found.{Environment.NewLine}Would you like to create a new report?");
+    }
+
+    string CreateReportOutputPath(string customerName)
+    {
+        var outputPath = Path.Join(Environment.CurrentDirectory,
+            $"{customerName}-{reportName}");
```
Comment (DavidBoike) [inline src/Tool/Commands/BaseCommand.cs:53]: Guard against illegal filename chars
```suggestion
        var customerFileName = Regex.Replace(customerName, @"[^\w\d]+", "-").Trim('-').ToLower()  
        var outputPath = Path.Join(Environment.CurrentDirectory,
            $"{customerFileName-{reportName}");
```

**Review by DavidBoike — CHANGES_REQUESTED** [review]

File: src/Tool/Commands/BaseCommand.cs [hunk 2]
```diff
@@ -9,25 +11,51 @@
...
+        var currentDirectory = new DirectoryInfo(Environment.CurrentDirectory);
+
+        var customerReport = $"{customerName}-{reportName}-{GenerateReportTimeStamp(DateTime.Now)}.json";
+
+        var files = currentDirectory.GetFiles($"{customerReport}*");
+
+        return files.Any();
+    }
+
+    bool ConfirmCreateNewReport()
```
[the remark this answers is not in the document]
Comment (awright18) [reply src/Tool/Commands/BaseCommand.cs:45]: This isn't looking for the same file, just any customer throughput file in the working directory. If you don't think that is necessary, then I'm happy to eliminate any code that cares about that.

[the remark this answers is not in the document]
Comment (DavidBoike) [reply src/Tool/Commands/BaseCommand.cs:44]: Yeah let's kill it in the name of simplicity. The files we create after this PR will be nice and sortable so they can grab the most recent one.

[the remark this answers is not in the document]
Comment (awright18) [reply src/Tool/Commands/BaseCommand.cs]: Removed

File: src/Tool/Commands/BaseCommand.cs [hunk 3]
```diff
@@ -9,25 +10,36 @@
...
+    {
+        return $"{dateTime.Year}{dateTime.Month}-{dateTime.Day}-{dateTime.Hour}-{dateTime.Minute}-{dateTime.Second}";
     }
 
-    public async Task Run(CancellationToken cancellationToken = default)
+    string CreateReportOutputPath(string customerName)
+    {
+        var customerFileName = Regex.Replace(customerName, @"[^\w\d]+", "-").Trim('-').ToLower();
+        var outputPath = Path.Join(Environment.CurrentDirectory,
+            $"{customerFileName}-{reportName}-{GenerateReportTimeStamp(DateTime.Now)}");
```
Comment (DavidBoike) [inline src/Tool/Commands/BaseCommand.cs:37]: Still need this - for example `dateTime.Day` won't produce leading zero and the result would not be sortable.
```suggestion
            $"{customerFileName}-{reportName}-{DateTime.Now:yyyyMMdd-HHmmss}");
```

File: src/Tool/Commands/BaseCommand.cs [hunk 4]
```diff
@@ -9,25 +10,36 @@
...
         if (!bool.TryParse(Environment.GetEnvironmentVariable("IS_DEVELOPMENT"), out isDevelopment))
         {
             isDevelopment = false;
         }
+    }
 
+    string GenerateReportTimeStamp(DateTime dateTime)
+    {
+        return $"{dateTime.Year}{dateTime.Month}-{dateTime.Day}-{dateTime.Hour}-{dateTime.Minute}-{dateTime.Second}";
     }
```
Comment (DavidBoike) [inline src/Tool/Commands/BaseCommand.cs:32]: Remove this method, see below.
```suggestion
```

**Review by DavidBoike — APPROVED** [review]

**Pull request merged**